2022-01-30 16:27:33,884 INFO    MainThread:54204 [wandb_setup.py:_flush():71] setting env: {'project': 'High_Income', 'run_group': 'dev'}
2022-01-30 16:27:33,884 WARNING MainThread:54204 [wandb_setup.py:_flush():71] could not save program above cwd: /Users/ivanovitchsilva/opt/anaconda3/envs/mlflow-646ab37a5ab6e97d711e16eeb4500ce1a8212984/bin/pytest
2022-01-30 16:27:33,884 INFO    MainThread:54204 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-30 16:27:33,884 INFO    MainThread:54204 [wandb_init.py:_log_setup():371] Logging user logs to /Users/ivanovitchsilva/high_income/check_data/wandb/run-20220130_162733-27hguizj/logs/debug.log
2022-01-30 16:27:33,885 INFO    MainThread:54204 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/ivanovitchsilva/high_income/check_data/wandb/run-20220130_162733-27hguizj/logs/debug-internal.log
2022-01-30 16:27:33,885 INFO    MainThread:54204 [wandb_init.py:init():404] calling init triggers
2022-01-30 16:27:33,885 INFO    MainThread:54204 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-30 16:27:33,886 INFO    MainThread:54204 [wandb_init.py:init():460] starting backend
2022-01-30 16:27:33,886 INFO    MainThread:54204 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-01-30 16:27:33,913 INFO    MainThread:54204 [backend.py:ensure_launched():216] starting backend process...
2022-01-30 16:27:33,929 INFO    MainThread:54204 [backend.py:ensure_launched():221] started backend process with pid: 54214
2022-01-30 16:27:33,931 INFO    MainThread:54204 [wandb_init.py:init():469] backend started and connected
2022-01-30 16:27:33,937 INFO    MainThread:54204 [wandb_init.py:init():533] updated telemetry
2022-01-30 16:27:33,967 INFO    MainThread:54204 [wandb_init.py:init():563] communicating current version
2022-01-30 16:27:34,650 INFO    MainThread:54204 [wandb_init.py:init():568] got version response 
2022-01-30 16:27:34,650 INFO    MainThread:54204 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-30 16:27:35,112 INFO    MainThread:54204 [wandb_init.py:init():606] starting run threads in backend
2022-01-30 16:27:36,151 INFO    MainThread:54204 [wandb_run.py:_console_start():1810] atexit reg
2022-01-30 16:27:36,151 INFO    MainThread:54204 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-30 16:27:36,151 INFO    MainThread:54204 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-30 16:27:36,152 INFO    MainThread:54204 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-30 16:27:36,152 INFO    MainThread:54204 [wandb_init.py:init():633] run started, returning control to user process
2022-01-30 16:27:41,258 INFO    MainThread:54204 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-30 16:27:41,259 INFO    MainThread:54204 [wandb_run.py:_restore():1752] restore
2022-01-30 16:27:44,204 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 1060
}

2022-01-30 16:27:44,761 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 1060
}

2022-01-30 16:27:45,169 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 4622
}

2022-01-30 16:27:45,270 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 4622
}

2022-01-30 16:27:45,377 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 4622
}

2022-01-30 16:27:45,483 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 4622
}

2022-01-30 16:27:45,589 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 4622
}

2022-01-30 16:27:45,695 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 4622
}

2022-01-30 16:27:45,802 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 4622
}

2022-01-30 16:27:45,908 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 4622
}

2022-01-30 16:27:46,016 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 4622
}

2022-01-30 16:27:46,117 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 4622
}

2022-01-30 16:27:46,224 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 4622
}

2022-01-30 16:27:46,332 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1060
  total_bytes: 4622
}

2022-01-30 16:27:46,439 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1670
  total_bytes: 4622
}

2022-01-30 16:27:46,541 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4622
  total_bytes: 4622
}

2022-01-30 16:27:46,651 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4622
  total_bytes: 4622
}

2022-01-30 16:27:46,759 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4622
  total_bytes: 4622
}

2022-01-30 16:27:46,863 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4622
  total_bytes: 4622
}

2022-01-30 16:27:46,970 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4622
  total_bytes: 4622
}

2022-01-30 16:27:47,077 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4622
  total_bytes: 4622
}

2022-01-30 16:27:47,317 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4622
  total_bytes: 4622
}

2022-01-30 16:27:48,300 INFO    MainThread:54204 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4622
  total_bytes: 4622
}
local_info {
}

2022-01-30 16:27:49,411 INFO    MainThread:54204 [wandb_run.py:_append_files():2180] logging synced files
